pkg/registry/core/pod/storage/storage.go

```diff
@@ -208,6 +219,32 @@ func (r *BindingREST) assignPod(ctx context.Context, podID string, machine strin
 	return
 }
 
+var _ = rest.Creater(&LegacyBindingREST{})
+
+// LegacyBindingREST implements the REST endpoint for binding pods to nodes when etcd is in use.
+type LegacyBindingREST struct {
```

Comment:
why was this needed? I thought BindingREST would be able to tell based on `len(name) == 0` whether it was being invoked as a subresource or a top-level resource

Reply:
actually, looks like the the name extraction in create.go will do the same thing this adapter is doing, and the name param will be populated... was there another reason LegacyBindingREST was created?

Reply:
mm... that triggers namedCreater, which triggers `includeName`, which returns an error if the scope.Namer.Name can't extract a name from the request